src/cpu/matmul/cpu_matmul_list.cpp

```diff
@@ -72,8 +72,9 @@ using namespace dnnl::impl::cpu::matmul;
 // clang-format off
 constexpr impl_list_item_t impl_list[] = REG_MATMUL_P({
 
-        CPU_INSTANCE_AARCH64(brgemm_matmul_t<sve_512>)
-        CPU_INSTANCE_AARCH64_ACL(acl_matmul_t)
+        CPU_INSTANCE_AARCH64(brgemm_matmul_t<sve_512>)        
+        CPU_INSTANCE_AARCH64_ACL(acl_matmul_t) 
+        CPU_INSTANCE_AARCH64(brgemm_matmul_t<sve_256>)       
```

Comment:
It would be nice if you could avoid templating implementation over isa. I know, it's not yet there for x64 matmul, but we are looking into it and will probably resolve that in v3.6. If you could avoid this path since the beginning, that would be good. It's fine to postpone this change to the next PR.

Reply:
Yes sure. We would keep this in mind during our further implementations.